### ISSUE TITLE
Online players doesn't get removed from game view

### ISSUE THREAD

**Issue by zars15** [CrescentCode/ValkyrieServer#6]
Online players get added properly, but they remain as online, even if they have left the game long time ago.

Also, the player names aren't retrieved properly, they appear as nil.

**Comment by Gskartwii** [comment]
Thanks for the report. I am aware of these bugs, and should get at least the latter on fixed shortly.

**Comment by Gskartwii** [comment]
I have finally verified that the above commit has fixed the player name issue. I couldn't before due to the server being down because of external reasons.

For the other issue, I have manually cleared the online players. The issue should be fixed server-side on the keepalive-tracking branch, but is waiting for Roblox-side changes to be done to the private player tracking module (which is private to prevent developers from spoofing play times)
